### ISSUE TITLE
Error msg when using "loopPlayback"

### ISSUE THREAD

**Issue by sanihaq** [erikas-taroza/simple_audio#11]
Error showing when repeating a file
```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method setPlaybackState on channel simple_audio)
#0      MethodChannel._invokeMethod
platform_channel.dart:313
<asynchronous suspension>
```

**Comment by erikas-taroza** [comment]
Ah that's only supposed to run for Android, macOS, and iOS. Let me fix it.

**Comment by erikas-taroza** [comment]
Fixed in v1.4.3
